pubspec.yaml

```diff
@@ -11,9 +11,9 @@ dependencies:
     sdk: flutter
 
   animated_stack_widget: ^0.0.4
-  flutter_map: ^1.0.0
+  flutter_map: ^2.0.0
```

Comment:
Same here

Reply:
flutter_map 2.1.1 is the latest version btw

Reply:
> flutter_map 2.1.1 is the latest version btw

That is within the range that I suggested above

Reply:
I don't think that's a solution. It's more of a workaround.